diff -ru ghdl-1.0.0/configure ghdl-1.0.0-new/configure
--- ghdl-1.0.0/configure	2021-02-02 12:52:50.000000000 -0700
+++ ghdl-1.0.0-new/configure	2021-08-29 12:57:23.929215512 -0600
@@ -31,7 +31,7 @@
 MAKE=${MAKE:-make}
 LDFLAGS=
 prefix=/usr/local
-libdirsuffix=lib/ghdl
+libdirsuffix=ghdl
 libdirreverse=../..
 gcc_src_dir=
 llvm_config=
@@ -48,11 +48,20 @@
 EXEEXT=
 SOEXT=.so
 PIC_FLAGS=-fPIC
+ac_tool_prefix=
+libdir=
+# Unused parameters
+mandir=
+infodir=
+datadir=
+sysconfdir=
+localstatedir=
+
 
 show_help=no
 progname=$0
 
-subst_vars="CC CXX GNATMAKE MAKE CFLAGS LDFLAGS build srcdir abs_srcdir prefix backend libdirsuffix libdirreverse gcc_src_dir llvm_config llvm_be backtrace_lib build_mode EXEEXT SOEXT PIC_FLAGS default_pic enable_werror enable_checks enable_gplcompat enable_libghdl libghdl_version ghdl_version"
+subst_vars="CC CXX GNATMAKE MAKE CFLAGS LDFLAGS build srcdir abs_srcdir prefix backend libdirsuffix libdirreverse gcc_src_dir llvm_config llvm_be backtrace_lib build_mode EXEEXT SOEXT PIC_FLAGS default_pic enable_werror enable_checks enable_gplcompat enable_libghdl libghdl_version ghdl_version ac_tool_prefix libdir mandir infodir datadir sysconfdir localstatedir"
 
 # Find srcdir
 srcdir=`dirname $progname`
@@ -111,6 +120,16 @@
     --default-pic)          default_pic=true;;
     --enable-coverage)      build_mode="coverage";;
     -h|-help|--help)        show_help=yes;;
+    --build=*)              build="$optarg";;
+    --host=*)               host="$optarg";;
+    --libdir=*)             libdir="$optarg";;
+    # Unused parameters
+    --mandir=*)             mandir="$optarg";;
+    --infodir=*)            infodir="$optarg";;
+    --datadir=*)            datadir="$optarg";;
+    --sysconfdir=*)         sysconfdir="$optarg";;
+    --localstatedir=*)      localstatedir="$optarg";;
+
     *)
       echo "$0: unknown option $opt; try $0 --help"
       exit 1
@@ -118,6 +137,14 @@
   esac
 done
 
+ac_tool_prefix=
+test -n "$host_alias" && ac_tool_prefix=$host_alias-
+
+if test "x$libdir" = "x"; then
+    libdir=$prefix/lib
+fi
+
+
 # Help
 if test $show_help != no; then
   cat <<EOF
@@ -275,6 +302,7 @@
        check_version  9.0 $llvm_version ||
        check_version 10.0 $llvm_version ||
        check_version 11.0 $llvm_version ||
+       check_version 11.1 $llvm_version ||
        false; then
     echo "Debugging is enabled with llvm $llvm_version"
   else
@@ -403,7 +431,7 @@
     -e "s%@COMPILER_LLVM@%ghdl1-llvm$EXEEXT%" \
     -e "s%@POST_PROCESSOR@%oread-$backend%" \
     -e "s%@INSTALL_PREFIX@%$prefix%" \
-    -e "s%@LIB_PREFIX@%$libdirsuffix%" \
+    -e "s%@LIB_PREFIX@%$libdir/$libdirsuffix%" \
     -e "s%@SOEXT@%$SOEXT%" \
     -e "s%@default_pic@%$default_pic%" \
     < $srcdir/src/ghdldrv/default_paths.ads.in > default_paths.ads
diff -ru ghdl-1.0.0/Makefile.in ghdl-1.0.0-new/Makefile.in
--- ghdl-1.0.0/Makefile.in	2021-02-02 12:52:50.000000000 -0700
+++ ghdl-1.0.0-new/Makefile.in	2021-08-29 12:59:10.534163323 -0600
@@ -13,12 +13,12 @@
 #
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <gnu.org/licenses>.
-CC=@CC@
-CXX=@CXX@
+CC=@ac_tool_prefix@@CC@
+CXX=@ac_tool_prefix@@CXX@
 build=@build@
 srcdir=@srcdir@
 abs_srcdir=@abs_srcdir@
-GNATMAKE=@GNATMAKE@
+GNATMAKE=@ac_tool_prefix@@GNATMAKE@
 MAKE=@MAKE@
 prefix=@prefix@
 backend=@backend@
@@ -46,7 +46,7 @@
 PWD?=$(shell pwd)
 DESTDIR=
 bindir=$(prefix)/bin
-libdir=$(prefix)/lib
+libdir=@libdir@
 incdir=$(prefix)/include
 MKDIR=mkdir
 LN=ln -s
@@ -56,7 +56,7 @@
 GRT_RANLIB=ranlib
 GHDL_DESC?=tarball
 
-VHDL_LIB_DIR=$(prefix)/$(libdirsuffix)
+VHDL_LIB_DIR=$(libdir)/$(libdirsuffix)
 
 ifeq "$(enable_checks)" "true"
  # Debug + checks
diff -ru ghdl-1.0.0/scripts/gcc/Make-lang.in ghdl-1.0.0-new/scripts/gcc/Make-lang.in
--- ghdl-1.0.0/scripts/gcc/Make-lang.in	2021-02-02 12:52:50.000000000 -0700
+++ ghdl-1.0.0-new/scripts/gcc/Make-lang.in	2021-08-29 12:58:05.951200321 -0600
@@ -103,6 +103,7 @@
 	echo "   Compiler_Mcode : constant String := \"\";" >> tmp-dpaths.ads
 	echo "   Compiler_Llvm  : constant String := \"\";" >> tmp-dpaths.ads
 	echo "   Post_Processor : constant String := \"\";" >> tmp-dpaths.ads
+	# TODO: Fix this!!!
 	echo "   Lib_Prefix     : constant String :=">> tmp-dpaths.ads
 	echo "     \"lib/ghdl\";" >> tmp-dpaths.ads
 	echo "   Shared_Library_Extension : constant String :=">> tmp-dpaths.ads
